### PR TITLE
switching to chakra UI

### PR DESCRIPTION
**Pull request by arvi18**

### CODE REVIEW

File: src/Components/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      textDecoration: "none",
+      bg: useColorModeValue("gray.400", "gray.700"),
+    }}
+    href={"#"}
+  >
+    {children}
+  </Link>
+);
+
+export default function Nav() {
```
Comment (Soham041201) [inline src/Components/Nav.jsx:36]: Use spaces

File: src/Components/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    _hover={{
+      textDecoration: "none",
+      bg: useColorModeValue("gray.400", "gray.700"),
+    }}
+    href={"#"}
+  >
+    {children}
+  </Link>
+);
+
```
Comment (Soham041201) [inline src/Components/Nav.jsx:35]: Use functional arrow components for consistency

File: src/Components/PostsActivitySection.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
+import { Flex, Text, Box, Icon, HStack } from "@chakra-ui/react";
+import { chat, thumbsUp, stack } from "../icons/icons";
+import Vector from "../icons/Vector.svg";
+
+function PostsActivitySection() {
```
Comment (Soham041201) [inline src/Components/PostsActivitySection.jsx:5]: Use arrow functional components for consistancy

File: src/Components/Nav.jsx (hunk 1 above)
Comment (arvi18) [reply src/Components/Nav.jsx:36]: How to, I use prettier autoformat

File: src/Components/PostsActivitySection.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            <img src={Vector} alt="Posts" />
+            <Text fontSize={"xl"}>Posts</Text>
+          </HStack>
+
+          <HStack>
+            <Icon as={stack} boxSize={"1.7rem"}></Icon>
+            <Text fontSize={"xl"}>Activity</Text>
+          </HStack>
+        </HStack>
+        {/* first post starts here */}
```
Comment (Soham041201) [inline src/Components/PostsActivitySection.jsx:20]: Use .map() function rather than doing it this way

File: src/Components/ProductSection.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import { ArrowForwardIcon } from "@chakra-ui/icons";
+import { Box, Circle, HStack, Icon, Text } from "@chakra-ui/react";
+import { productHunt } from "../icons/icons";
+
+function ProductSection() {
```
Comment (Soham041201) [inline src/Components/ProductSection.jsx:5]: Function component for consistancy

File: src/Components/ProfileDetails.jsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+          style={{
+            height: "150px",
+            width: "150px",
+            objectFit: "cover",
+            borderRadius: "50%",
+          }}
+        />
+        <VStack width={"80vw"}>
+          <HStack width="100%" spacing={6}>
+            <Text fontSize="3xl" fontWeight={650}>
```
Comment (Soham041201) [inline src/Components/ProfileDetails.jsx:20]: Please dont hard-code these values. Create a variable and then use it. Will be easier to integrate and show the data from the backend

File: src/Components/ProfileDetails.jsx [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+            @ashneer
+          </Text>
+          <Text width="100%" textAlign="start" fontSize="md" fontWeight={600}>
+            Ye sab doglapan hai | Shark @ Shark Tank India
+          </Text>
+          {/*  */}
+          <HStack
+            width="100%"
+            padding="1rem 0"
+            //  paddingLeft={"12.35rem"}
```
Comment (Soham041201) [inline src/Components/ProfileDetails.jsx:41]: Remove comments if not using

File: src/Components/ProfileDetails.jsx [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+              <Text fontSize="md">0 Following</Text>
+            </HStack>
+            <HStack gap="1rem">
+              <Icon
+                as={globe}
+                // color="red"
+              ></Icon>
+              <Icon as={mail}></Icon>
+              <Icon as={linkedIn}></Icon>
+              {/* standard way of creating svg icon */}
```
Comment (Soham041201) [inline src/Components/ProfileDetails.jsx:56]: Remove comments if not using

File: src/Components/Nav.jsx (hunk 2 above)
Comment (arvi18) [reply src/Components/Nav.jsx:35]: okay I'll change it in next commit

File: src/Components/PostsActivitySection.jsx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+        <Flex>
```
Comment (Soham041201) [inline src/Components/PostsActivitySection.jsx:21]: Create this as a different component

File: src/Components/PostsActivitySection.jsx (hunk 4 above)
Comment (arvi18) [reply src/Components/PostsActivitySection.jsx:20]: okay

File: src/Components/ProductSection.jsx [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <>
+      <Box padding={"1rem"}>
+        <HStack padding={"1rem"}>
+          <Icon boxSize={"2rem"} as={productHunt}></Icon>
+          <Text fontSize="2xl">Products</Text>
+        </HStack>
+
+        <HStack>
```
Comment (Soham041201) [inline src/Components/ProductSection.jsx:14]: Create this as a different component

File: src/Components/ProfileDetails.jsx [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+              Pitch
+            </Button>
+          </HStack>
+          <Text width="100%" textAlign="start" fontSize="sm">
+            @ashneer
+          </Text>
+          <Text width="100%" textAlign="start" fontSize="md" fontWeight={600}>
+            Ye sab doglapan hai | Shark @ Shark Tank India
+          </Text>
+          {/*  */}
```
Comment (Soham041201) [inline src/Components/ProfileDetails.jsx:37]: Remove this if not needed

File: src/Components/ProfileDetails.jsx (hunk 6 above)
Comment (arvi18) [reply src/Components/ProfileDetails.jsx:20]: I'll discuss with rest of dudes, we'll go with variables

File: src/Components/ProfileDetails.jsx (hunk 7 above)
Comment (arvi18) [reply src/Components/ProfileDetails.jsx:41]: yeah okay

File: src/Components/PostsActivitySection.jsx (hunk 9 above)
Comment (arvi18) [reply src/Components/PostsActivitySection.jsx:21]: yes till EOD

File: src/Components/Nav.jsx (hunk 1 above)
Comment (Soham041201) [reply src/Components/Nav.jsx:36]: Just code in different blocks

**Pull request merged**